src/main/java/com/gmail/yuramitryahin/service/RoleServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.gmail.yuramitryahin.service;
+
+import com.gmail.yuramitryahin.dao.RoleDao;
+import com.gmail.yuramitryahin.model.Role;
+import com.gmail.yuramitryahin.model.RoleType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RoleServiceImpl implements RoleService {
+    private RoleDao roleDao;
```

Comment:
```suggestion
    private final RoleDao roleDao;
```
remember about consistency in your code

Reply:
ok, fixed